pkg/cmd/admin/router/router.go

```diff
@@ -325,6 +325,28 @@ func generateLivenessProbeConfig(cfg *RouterConfig,
 	return probe
 }
 
+func generatePrometheusExporterContainer(cfg *RouterConfig) *kapi.Container {
+	switch cfg.Type {
+	case "haproxy-router":
+		return &kapi.Container{
+			Name:  "prometheus-exporter",
+			Image: "prom/haproxy-exporter:latest",
```

Comment:
Hrm - we can't assume this for Origin, because the image is outside our control.  The image has to be configurable somehow (either on invocation or afterwards, and we may have to make this opt in for now via the router command (--gather-metrics)